Packs/RubrikPolaris/Scripts/RubrikSonarSetIncidentSeverityUsingUserRiskLevel/RubrikSonarSetIncidentSeverityUsingUserRiskLevel.py

```diff
@@ -0,0 +1,73 @@
+"""RubrikSonarSetIncidentSeverityUsingUserRiskLevel Script for Cortex XSOAR (aka Demisto)."""
+
+from typing import Dict
+
+import demistomock as demisto
+from CommonServerPython import *
+
+INCIDENT_SEVERITY_INT_TO_NAME = {
+    1: 'Low',
+    2: 'Medium',
+    3: 'High'
+}
+
+''' COMMAND FUNCTION '''
+
+
+def set_incident_severity_using_risk_level_command(args: Dict[str, Any]) -> CommandResults:
+    """Set the incident severity using the provided risk levels.
+
+    :type args: ``Dict[str, Any]``
+    :param args: Arguments provided by user.
+
+    :rtype: ``CommandResults``
+    :return: Standard command result.
+    """
+    risk_levels: list = argToList(args.get('risk_levels'))
+
+    if not risk_levels:
+        return CommandResults(readable_output='No risk_levels specified to update the incident severity.')
```

Comment:
Let's make risk_levels required in the yml.

Reply:
No, this was intentional as while using in playbooks we don't want to halt the execution instead just want to show message and proceed ahead with further tasks.

Reply:
Can you add this as a comment?